Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gradle:jdk-alpine
+
+USER root
+
+COPY . ./
+RUN gradle installDist --stacktrace
+
+FROM openjdk:8-jre-alpine
+
+RUN mkdir -p /opt/analyzer
```

Comment:
It this command necessary? In the C# analyzer, I begin with specifying the WORKDIR, and then copy into that. That might make things a little bit easier.